src/lib/controller/storeCache.test.ts

```diff
@@ -0,0 +1,101 @@
+import { describe, expect, it, jest, afterEach } from "@jest/globals";
+import { fillStoreCache, sendUpdatesAndFlushCache } from "./storeCache";
+import { Operation } from "fast-json-patch";
+import { GenericClass, K8s, KubernetesObject } from "kubernetes-fluent-client";
+import { K8sInit } from "kubernetes-fluent-client/dist/fluent/types";
+
+jest.mock("kubernetes-fluent-client");
+
+describe("sendCache", () => {
+  const mockK8s = jest.mocked(K8s);
+
+  afterEach(() => {
+    jest.resetAllMocks();
+  });
+
+  it("should reject unsupported operations", () => {
+    expect(() => {
+      fillStoreCache({}, "capability", "unsupported" as "remove", { key: ["key"] }); // Type coercion for "unsupported" to verify exception occurs
+    }).toThrow("Unsupported operation: unsupported");
+  });
+
+  describe("when updates are sent to the PeprStore", () => {
+    it("should clear the cache", async () => {
+      mockK8s.mockImplementation(<T extends GenericClass, K extends KubernetesObject>() => {
+        return {
+          Patch: jest.fn().mockResolvedValueOnce(undefined as never),
+        } as unknown as K8sInit<T, K>;
+      });
+
+      const input: Record<string, Operation> = {
+        entry: { op: "remove", path: "/some/path" },
+        entry2: { op: "add", path: "some/path", value: "value" },
+      };
+      const result = await sendUpdatesAndFlushCache(input, "some namespace", "some name");
+      expect(result).toStrictEqual({});
+    });
+
+    it("should clear cache for Unprocessable Entity Errors (HTTP/422)", async () => {
+      mockK8s.mockImplementation(<T extends GenericClass, K extends KubernetesObject>() => {
+        return {
+          Patch: jest.fn().mockRejectedValueOnce({ status: 422 } as never),
+        } as unknown as K8sInit<T, K>;
+      });
+
+      const input: Record<string, Operation> = {
+        entry: { op: "remove", path: "/some/path" },
+        entry2: { op: "add", path: "some/path", value: "value" },
+      };
+      const result = await sendUpdatesAndFlushCache(input, "some namespace", "some name");
+      expect(result).toStrictEqual({});
+    });
```

Comment:
```suggestion
    });

```

NBD.  Just missing a line break between tests.

Reply:
Fixed, figured the linter would've caught that :/